Main/ClientApp/src/Authorization/Logout.js

```diff
@@ -59,7 +61,19 @@ export class Logout extends Component {
       return <div></div>;
     }
     if (!!message) {
-      return <PanelWideMessage text={message} />;
+      return (
```

Comment:
Personally. I'd rather just redirect the user to the home page without any message or button, he can log back in from there. But this is also fine, you be the judge

Reply:
redirect is better option, i'll change that